packages/web-client/src/helpers/share/space.ts

```diff
@@ -29,6 +29,8 @@ export function buildSpaceShare(s, storageId): Share {
       additionalInfo: null
     },
     permissions,
-    role
+    role,
+    expiration: s.expirationDate,
+    expires: s.expirationDate
```

Comment:
I had another look and the backend is returning the date in RFC3339 format.
If we convert it to a js Date here then `DateTime.fromJSDate(expires)` should work too.
```suggestion
    expires: new Date(s.expirationDate)
```